python/shotgun_fields/image_widget.py

```diff
@@ -279,24 +318,53 @@ def _replace_image(self):
         """
         self._upload_image()
 
-    def _scale_pixmap(self):
+    def _scale_pixmap(self, pixmap):
         """
         Scale the pixmap in preparation for display.
         """
-        scaled_pixmap = self._pixmap.scaled(
+        return pixmap.scaled(
```

Comment:
maybe do a quick test with a collection of different aspect ratio thumbs and see what it looks like when displayed side by side!